src/slider/slider.scss

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: ($iui-s + $iui-xxs);
+
+@mixin iui-slider-component-container {
+  display: inline-flex;
+
+  span.iui-left,
+  span.iui-right {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: 4px;
+    }
+  }
+
+  span.iui-left {
+    margin-right: $iui-s;
+    text-align: right;
+  }
+
+  span.iui-right {
+    margin-left: $iui-s;
+  }
+}
+
+@mixin iui-slider-container {
+  position: relative;
+  z-index: 1; // above ticks
+  width: 100%;
+  pointer-events: none;
+
+  .iui-tooltip {
+    display: none;
+  }
+
+  // container for components drawn over top of input control
+  .iui-slider-display-container {
+    top: 0;
+    position: absolute;
+    width: 100%;
+
+    .iui-track {
+      pointer-events: none;  // let pointer event through to underlying range control
+      position: absolute;
+      width: 100%;
+      height: $track-height;
+      margin-top: 12px;
+      @include themed {
+        background-color: t(iui-color-background-4);
+      }
+    }
+
+    .iui-slider-thumb {
+      outline: none;
+      display: block;
+      position: absolute;
+      pointer-events: all;
+      height: $thumb-height;
+      width: $thumb-height;
+      border-radius: 100%;
+      transform: translate((-$thumb-height - 2) * 0.5, $track-height + 1); // 2 and 1 to adjust for border width
+      z-index: 2;
+      transition: border $iui-speed-fast ease;
+      @include themed {
+        background: t(iui-color-background-1);
+        border: 1px solid t(iui-color-background-5);
+      }
+    }
+  }
+
+  &.iui-progress-slider {
+    .iui-slider-thumb:focus,
+    .iui-slider-thumb.iui-active {
+      ~ .iui-tooltip {
+        position: absolute;
+        display: block;
+        margin-top: -$iui-xl;
+        transform: translateX(-50%);
+      }
+    }
+
+    .iui-slider-display-container {
+      .iui-slider-progress {
+        pointer-events: none;
+        display: block;
+        position: absolute;
+        height: $track-height;
+        border-radius: $track-height;
+        left: 0;
+        top: 12px;
+        z-index: 1;
+        @include themed {
+          background-color: t(iui-color-foreground-primary);
+        }
+      }
+    }
+  }
```

Comment:
translate value is hard to understand..

Reply:
Not sure what this means?

Reply:
oh...comment was put on the wierd place.. now its on 67 line:
![image](https://user-images.githubusercontent.com/81580355/124883040-9d9ad780-dfd9-11eb-8799-4ed369b49a02.png)
we can replace the top with the top value, but need to check what to do with horizontal placemenet and how we would implement that in react (if needed to change?)

Reply:
I have removed the y adjustment and added a 'top' setting. The X adjustment is still necessary.